client/signup/steps/difm-design-picker/themes.tsx

```diff
@@ -14,6 +14,9 @@ const difmThemes: Design[] = [
 		template: 'russell',
 		theme: 'russell',
 		title: 'Russell',
+		recipe: {
```

Comment:
I think we don't need to update this file as it would be removed in https://github.com/Automattic/wp-calypso/pull/62381

Reply:
If this PR is merged (and deployed) first, it will break DIFM :')

Reply:
:(